src/labeler.spec.ts

```diff
@@ -0,0 +1,130 @@
+const {getFileLabel, fileStatus, labelType} = require('./labeler')
+
+describe('getFileLabel', () => {
+  describe('when a main page is added', () => {
+    it('should return new command label', () => {
```

Comment:
There's no reason to have a `describe` wrapping a single `it` statement, just if we're going to group multiple tests together in some way.

Having said, I think you could group tests around a common theme for several of these, like having `describe('applying pageEdit label')` or `describe('applying translation label')`, etc.